packages/dropbear/build.sh

```diff
@@ -15,13 +15,23 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" ac_cv_func_getpass=yes LIBS=-lreadline"
 # build a multi-call binary
 TERMUX_PKG_EXTRA_MAKE_ARGS="MULTI=1"
 
+termux_step_post_make_install() {
+    {
+        echo "#!$TERMUX_PREFIX/bin/sh"
+        echo "exec dbclient \"\${@}\""
```

Comment:
Why not symlink?

Reply:
Ok, I will fix this. Just forgot that dropbearmulti understands 'ssh' symlink.